packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -46,4 +46,179 @@ export const components: ThemeOptions["components"] = {
       },
     },
   },
+  MuiButton: {
+    defaultProps: {
+      disableElevation: true,
+      disableRipple: true,
```

Comment:
I'm still getting the ripple effect. I think this needs to be applied to `MuiButtonBase` instead: https://mui.com/material-ui/getting-started/faq/#how-can-i-disable-the-ripple-effect-globally